vmngclient/api/template_api.py

```diff
@@ -135,6 +173,8 @@ def attach(self, name: str, device: Device) -> bool:
                 }
             ]
         }
+        if is_edited:
+            payload["deviceTemplateList"][0]["isEdited"] = True  # type: ignore
```

Comment:
isn't that better?
```python
payload = {
    "deviceTemplateList": [
        {
            "templateId": template_id,
            "isEdited": is_edited,
            "device": [
                {
                    "csv-status": "complete",
                    "csv-deviceId": device.uuid,
                    "csv-deviceIP": device.id,
                    "csv-host-name": device.hostname,
                    "csv-templateId": template_id,
                }
            ],
        }
    ]
}
```

Reply:
This is what I suspected, however are we sure that "isEdited" = False is different from "isEdited" missing in request to vManage?